docs/pages/admin-guides/access-controls/access-graph/ssh-keys-scan.mdx

```diff
@@ -0,0 +1,204 @@
+---
+title: Discover insecure Access Patterns with Teleport Policy
```

Comment:
I think the title is too generic - it doesn't mention SSH at all, but the entire guide is about scanning for SSH keys

Reply:
I couldn't find a good title for this. 
`SSH key scanning` isn't a good description so I end up using `Discover insecure SSH access with Teleport Policy`